pkg/systemd/host.js

```diff
@@ -334,10 +335,40 @@ PageServer.prototype = {
         var pmcd_service = service.proxy("pmcd");
         var pmlogger_service = service.proxy("pmlogger");
         var pmlogger_promise;
+        var pmlogger_exists = false;
+
+        function refresh_pmlogger_state() {
+            /* HACK: The pcp packages on Ubuntu and Debian inclue SysV init
+             * scripts in /etc, which stay around when removing (as opposed to
+             * purging) the package. Systemd treats those as valid units, even
+             * if they're not backed by packages anymore.  Thus, and
```

Comment:
Just in case you have to push again: please unbreak this "Thus, and" sentence.

Reply:
Oops, thanks.